src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Text, TouchableOpacity, View} from 'react-native';
 import Tooltip from 'react-native-walkthrough-tooltip';
 import MemoIcQuestionMark from '../../../assets/icons/Ic_question_mark';
-import {normalize} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `fonts` from the `../../../utils/fonts` module. This is a good change as it allows for more consistent typography across the application by using predefined font styles.
```diff
- import {normalize} from '../../../utils/fonts';
+ import {fonts, normalize} from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->